docs/pages/guides/extending/_meta.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  "extending-a-world": "Extending a World Permissionlessly",
```

Comment:
hmm now that I am looking more closely at this, do we expect more than one page in this section?

otherwise we can continue calling this section/dir `extending-a-world` and just use `index.mdx` for the page name

Reply:
Yes, that's the reason I moved it like this. "Extending" is for when you have an existing world and need to change it. Coming soon, a guide on how to add a table to such a `World`.

Reply:
it's still "extending-a-world" so I would still opt to keep the directory name to match the old path, use index.mdx for the base page, then can add other pages to the section for additional details

Reply:
Fixed.

Reply:
Done, please rereview.